src/symbolize_unittest.cc

```diff
@@ -150,6 +181,9 @@ TEST(Symbolize, SymbolizeWithDemangling) {
 #      if !defined(_MSC_VER) || !defined(NDEBUG)
 #        if defined(HAVE___CXA_DEMANGLE)
   EXPECT_STREQ("Foo::func(int)", TrySymbolize((void*)(&Foo::func)));
+  // Very long functions can be truncated, but we should not crash or return null
+  // Also the result should start properly.
+  EXPECT_TRUE(0 == std::string( TrySymbolize( (void*)(&Foo::longParamFunc))).find("Foo::longParamFunc("));
```

Comment:
Constructing a temporary `std::string` is unnecessary. Use `std::strstr` instead.

Reply:
ok